src/Tes.RunnerCLI/scripts/rootHash.sh

```diff
@@ -0,0 +1,21 @@
+﻿#!/bin/bash
+
+filename=$1
+blocksize=4194304 # 4 MiB
```

Comment:
You could expose it as a parameter with a default value:

`blocksize=${2:-4194304}`

(note: the syntax is weird, it's not a negative number)